student/pom.xml

```diff
@@ -24,7 +24,7 @@
 	<properties>
 		<progman-client.version>3.0.1</progman-client.version>
 		<sb11-mna-client.version>3.0.0</sb11-mna-client.version>
-		<tds-exam-client.version>0.0.18</tds-exam-client.version>
+		<tds-exam-client.version>0.0.20-SNAPSHOT</tds-exam-client.version>
```

Comment:
I'll change this to point to the latest after the exam PR is merged and released